presto-hive/src/main/java/io/prestosql/plugin/hive/HiveSessionProperties.java

```diff
@@ -93,6 +93,7 @@
     private static final String TIMESTAMP_PRECISION = "timestamp_precision";
     private static final String PARQUET_OPTIMIZED_WRITER_ENABLED = "experimental_parquet_optimized_writer_enabled";
     private static final String DYNAMIC_FILTERING_PROBE_BLOCKING_TIMEOUT = "dynamic_filtering_probe_blocking_timeout";
+    private static final String VIEWFS_TEMPORARY_STAGING_DIRECTORY_PATH = "../.hive-staging";
```

Comment:
The value `../.hive-staging` should be property name (not value).

Reply:
Also, we should add a test for viewfs. 
#1305

Reply:
Updated.

Reply:
@ebyhr yes, good idea to have a test